src/shapes/image.class.js

```diff
@@ -349,7 +357,13 @@
         if (element.toDataURL) {
           return element.toDataURL();
         }
-        return element.src;
+
+        if (this.srcFromAttribute && typeof element.getAttribute === 'function') {
```

Comment:
We support ie10 ( sort of ) ie11 and then modern browsers.
Can you verify if getAttribute is really unsupported from some of those?

Reply:
```suggestion
        if (this.srcFromAttribute) {
```